app/ui/src/main/java/com/fsck/k9/ui/settings/AboutActivity.kt

```diff
@@ -100,29 +102,42 @@ class AboutActivity : K9Activity() {
-                "Commons IO" to "http://commons.apache.org/io/",
-                "Glide" to "https://github.com/bumptech/glide",
-                "HoloColorPicker" to "https://github.com/LarsWerkman/HoloColorPicker",
-                "jsoup" to "https://jsoup.org/",
-                "jutf7" to "http://jutf7.sourceforge.net/",
-                "JZlib" to "http://www.jcraft.com/jzlib/",
-                "Mime4j" to "http://james.apache.org/mime4j/",
-                "Moshi" to "https://github.com/square/moshi",
-                "Okio" to "https://github.com/square/okio",
-                "SafeContentResolver" to "https://github.com/cketti/SafeContentResolver",
-                "ShowcaseView" to "https://github.com/amlcurran/ShowcaseView",
-                "Timber" to "https://github.com/JakeWharton/timber",
-                "TokenAutoComplete" to "https://github.com/splitwise/TokenAutoComplete/")
+        private val USED_LIBRARIES = arrayOf(
+                Library("Android Support Library", "https://developer.android.com/topic/libraries/support-library/index.html", "Apache License, Version 2.0"),
+                Library("Android-Support-Preference-V7-Fix", "https://github.com/Gericop/Android-Support-Preference-V7-Fix", "Unlicense/Apache License, Version 2.0"),
+                Library("ckChangeLog", "https://github.com/cketti/ckChangeLog", "Apache License, Version 2.0"),
+                Library("Commons IO", "https://commons.apache.org/io/", "Apache License, Version 2.0"),
+                Library("Glide", "https://github.com/bumptech/glide", "Mixed License"),
+                Library("HoloColorPicker", "https://github.com/LarsWerkman/HoloColorPicker", "Apache License, Version 2.0"),
+                Library("jsoup", "https://jsoup.org/", "MIT License"),
+                Library("jutf7", "http://jutf7.sourceforge.net/", "MIT License"),
+                Library("JZlib", "http://www.jcraft.com/jzlib/", "BSD-style License"),
+                Library("Mime4j", "http://james.apache.org/mime4j/", "Apache License, Version 2.0"),
+                Library("Moshi", "https://github.com/square/moshi", "Apache License, Version 2.0"),
+                Library("Okio", "https://github.com/square/okio", "Apache License, Version 2.0"),
+                Library("SafeContentResolver", "https://github.com/cketti/SafeContentResolver", "Apache License, Version 2.0"),
+                Library("ShowcaseView", "https://github.com/amlcurran/ShowcaseView", "Apache License, Version 2.0"),
+                Library("Timber", "https://github.com/JakeWharton/timber", "Apache License, Version 2.0"),
+                Library("TokenAutoComplete", "https://github.com/splitwise/TokenAutoComplete/", "Apache License, Version 2.0"))
 
         fun start(context: Context) {
             val intent = Intent(context, AboutActivity::class.java)
             context.startActivity(intent)
         }
     }
 }
+
+public class LibraryFragment : Fragment() {
```

Comment:
Converted to `RecyclerView` in force-pushed a1195f5
I used fragments because I knew they were a thing. I'm new to Android development - thanks for pointing me to that